README.md

```diff
@@ -1,2 +1,144 @@
-# hubverse-transform
-Data transform functions for hubverse model-output files
+# Hubverse Transform
+A package to perform data transformations on hubverse model-output files.
+
+The package contains a `ModelOutputHandler` class that reads, transforms, and writes a single Hubverse-compliant model-output file.
+
+Currently, its primary purpose is for use as an AWS Lambda function that transforms model-output files uploaded to hub S3 bucket.
+
+## Usage
+
+To install this package:
+
+```bash
+pip install git+https://github.com/Infectious-Disease-Modeling-Hubs/hubverse-transform.git
+```
+
+Sample usage:
+
+```python
+from hubverse_transform.model_output import ModelOutputHandler
+
+# to use with a local model-output file
+
+mo = ModelOutputHandler(
```

Comment:
This example is awkward because while you _can_ instantiate the class like this, it's not what I focused on (the lambda function uses the `from_s3` method to create the class instance.

We can come back and create a better sample usage snippet once the example hub data is on the cloud.